lldb/tools/lldb-dap/README.md

```diff
@@ -18,9 +18,10 @@
 
 # Introduction
```

Comment:
I think a stub README at the old lldb-vscode location is worth adding. It should just link here.

Reply:
I've made `lldb-vscode` a symlink to `lldb-dap`.